lib/SourceMapDevToolPlugin.js

```diff
@@ -12,11 +12,6 @@ var SourceMapDevToolModuleOptionsPlugin = require("./SourceMapDevToolModuleOptio
 function SourceMapDevToolPlugin(options) {
 	if(arguments.length > 1)
 		throw new Error("SourceMapDevToolPlugin only takes one argument (pass an options object)");
-	if(typeof options === "string") {
```

Comment:
in fact, here has not backward compatibility. Because options is redefined as
```
options = {		
￼  	sourceMapFilename: options		
};
```
`options.sourceMapFilename` is never used afterwards.